packages/components/psammead-leading-story-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -127,9 +139,11 @@ exports[`LeadingStoryPromo should render RTL leading story promo correctly 1`] =
 
 <div
   class="c0"
+  dir="ltr"
```

Comment:
In a test "LeadingStoryPromo should render RTL leading story promo correctly" you should pass `dir="rtl"` since right now in a snapshot for RTL there is a wrong direction

Reply:
Good spot! Should be done now.